packages/client/machines/appMachine.ts

```diff
@@ -453,6 +456,50 @@ export function createAppMachine({
                             },
                         },
 
+                        resendingConfirmationEmail: {
```

Comment:
Thoughts: This means that even if the user has confirmed her email or user is not on email confirmation screen he could ask for email resend ?

Reply:
`resendingConfirmationEmail` state is at the same level  as `formHandler` and `pollingHandler` states inside `waitingForUserEmailConfirmation` state. I think what you describe can not happen 🤔 Do I miss something?

Reply:
Ah my bad !